test/integration/docker-compose-localbuild.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
```

Comment:
I think it would be a good idea to put a remark here on what this extension does. Pointing out that this mounts the current backend in order to test it would for sure help others that come across this code.